src/main/java/seedu/address/ui/DeadlinePanel.java

```diff
@@ -30,6 +30,9 @@ public DeadlinePanel(Deadline deadline, int displayedIndex) {
         this.deadline = deadline;
         id.setText(displayedIndex + ". ");
         task.setText(deadline.getTask().toString());
```

Comment:
You can change the task from label to Textflow so that the text will not be concatenated and you can change the color of the text.

Reply:
Nice, thanks!